server/QuestionsAndAnswers/questions.js

```diff
@@ -0,0 +1,56 @@
+var routes = require('express').Router();
+var axios = require('axios');
+var url = require('url');
+var config = require('../../config');
+var endPoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-atx/qa/questions';
+
+var auth = { headers: { 'Authorization': `${config}` }};
+
+routes.get('/', (req, res) => {
+  axios.get(endPoint, { headers: { 'Authorization': `${config}` }, params: { product_id: req.query.product_id } })
```

Comment:
you have `auth` defined on line `7`, may make sense to use that here in line `10`

Reply:
Oh yes, i learned about the use of the spread operator on objects after this part. I'll apply it hear like :
{...auth, params:{}}